hack/kokoro-test.sh

```diff
@@ -22,7 +22,35 @@ echo "Running unit tests..."
 ./test.sh
 
 echo "Running integration tests..."
+
+# Run these in parallel.
+
 # Generate the integration test yaml and pass it to cloud build via stdin.
-python ftl/ftl_node_integration_tests_yaml.py | gcloud container builds submit --config /dev/fd/0 .
-python ftl/ftl_php_integration_tests_yaml.py | gcloud container builds submit --config /dev/fd/0 .
-gcloud container builds submit --config ftl/ftl_python_integration_tests.yaml .
+python ftl/ftl_node_integration_tests_yaml.py | gcloud container builds submit --config /dev/fd/0 . > node.log &
```

Comment:
at the language level, the test themselves have lot of parallel of pieces that I believe currently run sequentially:
gcp_build_test, packages_test, packages_lock_test, destination_test